packages/theme/package.json

```diff
@@ -43,8 +43,8 @@
   "dependencies": {
     "@oclif/core": "3.26.5",
     "@shopify/cli-kit": "3.69.0",
-    "@shopify/theme-check-node": "2.9.0",
-    "@shopify/theme-language-server-node": "1.12.1",
```

Comment:
Ive seen changesets created for bumps like these. Should we have it here too?

Reply:
Good callout, I'll add one!